articles/custom_storage_location.md

```diff
@@ -222,6 +222,117 @@ f <- synStore(f)
 
 Please see the [REST docs](http://docs.synapse.org/rest/org/sagebionetworks/repo/model/project/ExternalS3StorageLocationSetting.html){:target="_blank"} for more information on setting external storage location settings using our REST API.
 
+## Setting Up an External Google Cloud Storage Bucket
+
+Follow the documentation on Google Cloud's site to **[Create a Bucket](https://cloud.google.com/storage/docs/creating-buckets){:target="_blank"}**. 
+
```

Comment:
The link and the button go to same page, should remove one?

Reply:
Was just duplicating AWS instructions, but I removed it.